synapse/module_api/callbacks/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 The Matrix.org Foundation C.I.C.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .account_validity_callbacks import AccountValidityModuleApiCallbacks
```

Comment:
I think we usually use absolute imports, but 🤷

Reply:
Hmm, indeed. There are a bunch of examples of us using relative imports as well, but I think there's a general preference for absolute imports in case files are moved.

No need to further the bad habit, so I've updated in 449d01d0b.